fastlane/metadata/android/fr/full_description.txt

```diff
@@ -0,0 +1,18 @@
+Vous obtiendrez excactement ce à quoi vous vous attendez:
+Une jolie boussole simple pointant vers le nord magnétique
+
+Features:
+* Pointe le nord magnétique
+* Montre les degré
+You’ll get exactly what you expect.
```

Comment:
I think there a still some lines from the english description

Reply:
you're right 👍, corrected now